sycl/ReleaseNotes.md

```diff
@@ -1,3 +1,1595 @@
+  to return new `unknown` enumerator if device architecture cannot be properly
+  detected. intel/llvm#14077
+- Clarified [`sycl_ext_intel_device_info`](https://github.com/intel/llvm/blob/ebb3b4a21b3b0e977f44434781729df7de83e436/sycl/doc/extensions/supported/sycl_ext_intel_device_info.md)
+  extension to specify which exact exceptions are being thrown on errors. intel/llvm#14576
+- Introduced versioning and release process
+  [documentation](https://github.com/intel/llvm/blob/ebb3b4a21b3b0e977f44434781729df7de83e436/sycl/doc/syclcompat/README.md#versioning)
+  for SYCLcompat. intel/llvm#14457
+- Extended our
+  [contribution guidelines](https://github.com/intel/llvm/blob/ebb3b4a21b3b0e977f44434781729df7de83e436/sycl/doc/developer/ContributeToDPCPP.md#unified-runtime-updates)
+  to document update process for Unified Runtime component.
+- Updated
+  [`sycl_ext_oneapi_work_group_static`](https://github.com/intel/llvm/blob/ebb3b4a21b3b0e977f44434781729df7de83e436/sycl/doc/extensions/proposed/sycl_ext_oneapi_work_group_static.asciidoc)
+  extension (it is not `work_group_specific` anymore). intel/llvm#14271
+- Updated
+  [`sycl_ext_oneapi_matrix`](https://github.com/intel/llvm/blob/ebb3b4a21b3b0e977f44434781729df7de83e436/sycl/doc/extensions/experimental/sycl_ext_matrix/sycl_ext_oneapi_matrix.asciidoc)
+  extension:
+  - listed 1x64x16 `bfloat16` matrix combination as available on PVC.
+    intel/llvm#13587
+  - clarified the meaning of `joint_matrix_prefetch` template arguments.
+    intel/llvm#13796
+  - added a note about known issue with some CUDA devices. intel/llvm#14178
+- Added revision 2 of
+  [`sycl_ext_intel_matrix`](https://github.com/intel/llvm/blob/ebb3b4a21b3b0e977f44434781729df7de83e436/sycl/doc/extensions/experimental/sycl_ext_matrix/sycl_ext_intel_matrix.asciidoc)
+  extension which introduces load, store and fill matrix operations with
+  out-of-bounds checks. intel/llvm#11172
+- Updated
+  [`sycl_ext_oneapi_group`](https://github.com/intel/llvm/blob/ebb3b4a21b3b0e977f44434781729df7de83e436/sycl/doc/extensions/experimental/sycl_ext_oneapi_graph.asciidoc)
+  extension:
+  - added new functionality to update arguments and ND-range sizes of kernel
+    nodes within a graph. intel/llvm#12486
+  - clarified `enable_profiling` property in `command_graph::finalize`
+    intel/llvm#14067
+  - clarified how graph edges are recorded in a graph with in-order queues.
+    intel/llvm#12916
+  - clarified how code within command-group functions is handled.
+    intel/llvm#13699
+  - clarified interaction between queue properties and graphs. intel/llvm#13681
+  - added `enable_profiling` property. intel/llvm#13088
+- Updated
+  [`sycl_ext_oneapi_bindless_images`](https://github.com/intel/llvm/blob/ebb3b4a21b3b0e977f44434781729df7de83e436/sycl/doc/extensions/experimental/sycl_ext_oneapi_bindless_images.asciidoc)
+  extension:
+  - added support external semaphores that can take value. intel/llvm#13860
```

Comment:
```suggestion
  - added support external semaphores that can take a value. intel/llvm#13860
```

Reply:
Applied in 08997d4314baab9d10f3b3bdc68b394ae49a40d6